WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -163,22 +164,14 @@ private extension OrderTableViewCell {
 
 private extension OrderTableViewCell {
     enum Localization {
-        static func title(orderNumber: String, firstName: String, lastName: String) -> String {
-            let format = NSLocalizedString("#%1$@ %2$@ %3$@", comment: "In Order List,"
-                + " the pattern to show the order number and the full name. For example, “#123 John Doe”."
-                + " The %1$@ is the order number. The %2$@ is the first name. The %3$@ is the last name.")
-
-            return String.localizedStringWithFormat(format, orderNumber, firstName, lastName)
-        }
-
-        static func title(orderNumber: String) -> String {
+        static func title(orderNumber: String, customerName: String) -> String {
             let format = NSLocalizedString("#%@ %@", comment: "In Order List,"
                 + " the pattern to show the order number. For example, “#123456”."
                 + " The %@ placeholder is the order number.")
 
-            let guestName: String = NSLocalizedString("Guest", comment: "In Order List, the name of the billed person when there are no name and last name.")
-
-            return String.localizedStringWithFormat(format, orderNumber, guestName)
+            return String.localizedStringWithFormat(format, orderNumber, customerName)
         }
+
+        static let guestName = NSLocalizedString("Guest", comment: "In Order List, the name of the billed person when there are no name and last name.")
```

Comment:
super nit suggestion:

```suggestion
        static let guestName = NSLocalizedString("Guest", comment: "In Order List, the name of the billed person when there are no first and last name.")
```